<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-A0L6HGC" ignored="6" name="Default test" tests="1" failures="1" timestamp="30 Dec 2020 14:40:53 GMT" time="34.897" errors="0">
  <testcase name="verifyArrowButtonWorking" time="15.585" classname="com.test.MoeInnovationCell.MoeInnovationTest">
    <failure type="java.lang.AssertionError" message="Condition is:=false expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Condition is:=false expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at com.web.page.WebAction.assertTrue(WebAction.java:1255)
at com.test.MoeInnovationCell.MoeInnovationTest.verifyArrowButtonWorking(MoeInnovationTest.java:227)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyArrowButtonWorking -->
  <testcase name="verifyPresenceOfHeaderAndContents" time="0.0" classname="com.test.MoeInnovationCell.MoeInnovationTest">
    <ignored/>
  </testcase> <!-- verifyPresenceOfHeaderAndContents -->
  <testcase name="clickOnLoginButtonAndVerifySignUpPage" time="0.0" classname="com.test.MoeInnovationCell.MoeInnovationTest">
    <ignored/>
  </testcase> <!-- clickOnLoginButtonAndVerifySignUpPage -->
  <testcase name="verifyFooterLink" time="0.0" classname="com.test.MoeInnovationCell.MoeInnovationTest">
    <ignored/>
  </testcase> <!-- verifyFooterLink -->
  <testcase name="clickOnLoginButtonAndVerifyLoginPage" time="0.0" classname="com.test.MoeInnovationCell.MoeInnovationTest">
    <ignored/>
  </testcase> <!-- clickOnLoginButtonAndVerifyLoginPage -->
  <testcase name="verifyPresenceOfMoeLogo" time="0.0" classname="com.test.MoeInnovationCell.MoeInnovationTest">
    <ignored/>
  </testcase> <!-- verifyPresenceOfMoeLogo -->
  <testcase name="verifyMapIsPresentAndAllRegions" time="0.0" classname="com.test.MoeInnovationCell.MoeInnovationTest">
    <ignored/>
  </testcase> <!-- verifyMapIsPresentAndAllRegions -->
</testsuite> <!-- Default test -->
